### ISSUE TITLE
Package request: tdarr

### ISSUE THREAD

**Issue by camilosampedro** [NixOS/nixpkgs#273459]
**Project description**

As defined by themselves in [their website](https://docs.tdarr.io/docs/welcome/what):
> Tdarr is a popular conditional transcoding application for processing large (or small) media libraries. The application comes in the form of a click-to-run web-app, which you run on your own device and access through a web browser.

PKGBUILD from AUR if it helps: https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=tdarr

**Metadata**

* homepage URL: https://home.tdarr.io/
* source URL: https://github.com/HaveAGitGat/Tdarr
* license: [ISC](https://github.com/HaveAGitGat/Tdarr/blob/e67afc73532ba01939d67a720c09900a1e1ae898/package.json#L12)
* platforms: It has binaries for: linux, darwin, windows ([Downloads page](https://home.tdarr.io/download/))

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

**Comment by camilosampedro** [comment]
I'm a bit confused with the license of this package. There are inconsistencies:
- The homepage has a license that looks a bit restrictive: https://home.tdarr.io/eula
- The repository shows that the project uses an ISC license, with is much more permissive: https://en.wikipedia.org/wiki/ISC_license
- In the AUR, the configs say that it's a package under the GPL3 license: https://aur.archlinux.org/cgit/aur.git/tree/.SRCINFO?h=tdarr

Better to clarify this before adding it

**Comment by eclairevoyant** [comment]
Scripts in the are AUR are from random people who generally do not pay attention to licensing; that should not be relevant here. As per HaveAGitGat/Tdarr#743, it's unfree.

**Comment by camilosampedro** [comment]
True, I've asked them in HaveAGitGat/Tdarr#941 to provide a single license. But you are right, it looks like it's unfree.

**Comment by Thiago-Assis-T** [comment]
since now it got a licence, how long will it take to package the application? and if i can be of assistance, let me know, beware, i don't know how to package stuff in nix and i'm still a noob, but willing to help however i can, be it to try and build or provide logs

**Comment by Peronia** [comment]
Hi,
I just started to use nix. I added my try of the tdarr server at the PR. However, I failed because `fetchurl` is always undefined. Any help is welcome (:
